modules/clock_info.js

```diff
@@ -48,6 +49,15 @@ example.clkinfo.js :
 
 */
 
+let storage = require("Storage");
+let stepGoal = undefined;
+// Load step goal from health app and pedometer widget
+let d = storage.readJSON("health.json", true) || {};
```

Comment:
Thanks! Maybe we should make `wpedom` use `health.json` for this as well? Seems like a shame if apps have to pull this from two places

Reply:
Definitely. Maybe also document it somewhere (besides the Health's readme) so that everyone would that instead of implementing their own settings?

Reply:
Yes, sounds like a good plan! Just filed an issue for this: https://github.com/espruino/BangleApps/issues/2318